plotting/plot_factory/plot_factory.py

```diff
@@ -0,0 +1,208 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Constructs a plot and DashApp object for insertion into directly into a web page
+"""
+
+# Note pandas and plotly.graph_objs dependencies are used, but not recognised by pycharm interpreter
+
+# Visualisation Dependencies
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objects as go  # pylint: disable=unused-import
```

Comment:
`go` is unused import?

Reply:
This dependency is used as noted in docstring. Not recognised by IDE due to being called in eval